tools/find_untranslated_strings.py

```diff
@@ -0,0 +1,159 @@
+#! /usr/bin/env python
+# -*- coding: utf-8 -*-
+from __future__ import print_function
+from collections import namedtuple
+
+import re
+
+TransAndKey = namedtuple("TransAndKey", "translation, key")
+
+
+translation = re.compile(r'([a-z]{2}|zh-Han[st])\s*=\s*.*$', re.S | re.MULTILINE)
+
+ITUNES_LANGS = ['en', 'ru', 'ar', 'cs', 'da', 'nl', 'fi', 'fr', 'de', 'hu', 'id', 'it', 'ja', 'ko', 'nb', 'pl', 'pt', 'ro', 'sl', 'es', 'sv', 'th', 'tr', 'uk', 'vi', 'zh-Hans', 'zh-Hant']
+
+
+class StringsTxt:
+
+    def __init__(self):
+        self.translations = {} # dict<key, dict<lang, translation>>
+        self.comments_and_tags = {}
+        self.with_english = []
+        self.current_key = ""
+        self.all_langs = set()
+        self.duplicates = {} # dict<lang, TransAndKey>
+        self.keys_in_order = []
```

Comment:
For me, keys_in_order sounds quite strange, because there can be a lot of different ordering relations. If you want this name, keep it, but for me it's obfuscating. Plain list of keys looks good for me.